easyApp/Logic/Maintenance.py

```diff
@@ -189,27 +200,33 @@ def _getReleaseNotes(self):
         release_notes = re.sub(r'^# ', "### ", release_notes)
         return release_notes
 
+    def _getWebDate(self):
+        web_changelog = self._getWebChangelog()
+        pattern = r'^# Version .*? \(([A-Za-z0-9\s]*)\)'
+        matches = re.findall(pattern, web_changelog)
+        web_date = matches[0]
+        return web_date
+
```

Comment:
`QApplication.quit()` in `installUpdate` in line 70 will need user confirmation if state is changed.